core/validation/utils.py

```diff
@@ -12,3 +17,95 @@ def remove_duplicate_indexes(df: pd.DataFrame):
     :return: the DataFrame without duplicate indexes
     """
     return df[~df.index.duplicated(keep="first")]
+
+
+def get_start_date_column_for_failures(failed_rows: pd.DataFrame) -> list[tuple] | None:
+    """
+    Retrieves the 'Start_Date' column values for the specified indexes.
+
+    :param failed_rows: A pandas DataFrame containing the data of rows that have failed validation.
+    :return: A list of tuples, where each tuple contains an index and the corresponding 'Start_Date' value.
+    """
+    start_dates_series = failed_rows["Start_Date"] if "Start_Date" in failed_rows.columns else None
+    if start_dates_series is not None:
```

Comment:
Can we simplify ```if start_dates_series is not None:``` to ```if start_dates_series:``` in this case

Reply:
Unfortunately not as you cannot assert a truth value about a DataFrame or a Series.